File: graziela/Modulo3/Aula5/Exercicio4.py
'''
Faça um programa que leia uma matriz 3x3 de inteiros e retorne a
linha de maior soma. Imprima na tela a matriz, a linha de maior soma e a soma.
'''

matriz = []
maiorSoma = 0
linhaSoma = 0

for linha in range(3):
    linhaMatriz = []
    soma = 0
    for coluna in range(3):
        linhaMatriz.append(int(input('Digite um número: ')))
        soma = soma + linhaMatriz[coluna]
    matriz.append(linhaMatriz)
    if soma > maiorSoma:
        maiorSoma = soma
        linhaSoma = linha

print('Matriz: ')
print(matriz)
print('Maior soma: ', maiorSoma)
print('Linha da maior soma: ', linhaSoma)

File: graziela/Modulo4/AulaJava/src/Aula5/Exercicio4.java
package Aula5;

import javax.swing.JOptionPane;

public class Exercicio4 {
	public static void main(String[] args) {
		/*
		 * Escreva um programa que contenha 2 matriz 3 x 3 e uma matriz resultante
		 * da multiplicação das 2 matrizes anteriores.
		 */
		int matriz1[][] = new int[3][3];
		int matriz2[][] = new int[3][3];
		int matriz3[][] = new int[3][3];

		for (int linha = 0; linha < matriz1.length; linha++) {
			for (int coluna = 0; coluna < matriz1[linha].length; coluna++) {
				matriz1[linha][coluna] = Integer.parseInt(JOptionPane.showInputDialog("Digite um número[" + linha + "][" + coluna + "]:"));
			}
		}

		String resultado = "Resultado da multiplicação: ";
		for (int linha = 0; linha < matriz2.length; linha++) {
			for (int coluna = 0; coluna < matriz2[linha].length; coluna++) {
				matriz2[linha][coluna] = Integer.parseInt(JOptionPane.showInputDialog("Digite um número[" + linha + "][" + coluna + "]:"));
				matriz3[linha][coluna] = matriz1[linha][coluna] * matriz2[linha][coluna];
				resultado += "\n[" + linha + "][" + coluna + "] -> " + matriz1[linha][coluna]
							+ " * " + matriz2[linha][coluna] + " = " + matriz3[linha][coluna];
			}
		}

		JOptionPane.showMessageDialog(null, resultado);
	}
}
